common/src/main/java/org/apache/uniffle/common/RemoteStorageInfo.java

```diff
@@ -34,8 +34,10 @@ public class RemoteStorageInfo implements Serializable {
   private final String path;
   private final Map<String, String> confItems;
 
+  private String user;
```

Comment:
user shouldn't be member of RemoteStorageInfo which is used for describe storage only.

Reply:
You are right. It looks the RemoteStorageInfo should only store storage metadata instead of spark job info like user.

Maybe we should still add the param of `user` in the method of `registerShuffle`. Do u think so?

Reply:
@zuston I'm ok with add the param of `user` in the method of `registerShuffle`